sw/airborne/firmwares/rotorcraft/autopilot_arming_yaw.h

```diff
@@ -56,12 +57,14 @@ enum arming_state {
 
 uint32_t autopilot_motors_on_counter;
 enum arming_state autopilot_check_motor_status;
+float motor_kill_time;
 
 
 static inline void autopilot_arming_init(void)
 {
   autopilot_motors_on_counter = 0;
   autopilot_check_motor_status = STATUS_INITIALISE_RC;
+  motor_kill_time = 0.0;
```

Comment:
this means that the first 10 seconds after booting, the kill can arm? Can it be set to -10seconds?

Reply:
No normally not, as you can only re-arm from "STATUS_MOTORS_AUTOMATICALLY_OFF", and you can only get there once you're armed. I'm fine with putting -10 though.